### ISSUE TITLE
Cross entropy loss gradient

### ISSUE THREAD

**Issue by victorhcm** [seyedamo/cs231n#1]
Hi Seyedamo, 

I have recently started doing the course and I'm currently trying to compute the gradient of the cross entropy loss for the Softmax function. I was able to find the gradient w.r.t. [$f_{y_i}$](https://github.com/tuan3w/cs231n/blob/master/assignment1/cs231n/classifiers/softmax.py#L39), but I'm having trouble to compute the gradient with regarding [$f_{j}$](https://github.com/tuan3w/cs231n/blob/master/assignment1/cs231n/classifiers/softmax.py#L37). In your gradient, you have found $\frac{e^{f_j}}{\sum_j{e^{f_j}}} \* x_i$, while I'm finding only $x_i$.

How have you done it? :)

Thanks!

**Comment by victorhcm** [comment]
Hey, @seyedamo, in another repository I was pointed to [a page](http://cs231n.github.io/neural-networks-case-study/) that explains it. :)
